modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1233,6 +1233,39 @@ Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 
 Topology notifications provide additional information related to managing databases and servers.
 
+[#_neo_clientnotification_cluster_cordonedserversexistedduringallocation]
+=== CordonedServersExistedDuringAllocation
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.CordonedServersExistedDuringAllocation
+|Title
+a| Cordoned servers existed when making an allocation decision.
+|Severity
+m|WARNING
+|Category
+m|TOPOLOGY
+|===
+
+.Cordoned servers existed during an allocation decision
+====
+This notification will be returned by any command that triggers an allocation decision. Examples are `CREATE DATABASE`, `ALTER DATABASE`, `DEALLOCATE DATABASES FROM SERVER`, `ALTER DATABASE`.
+Note, `REALLOCATE DATABASES` requires there to be no cordoned servers to be executed successfully thus will never not return this notification.
+
+For example, consider a cluster with three servers, of which server `123e4567-e89b-12d3-a456-426614174000` has been cordoned using the `dbms.cluster.cordonServer` procedure. Then the below command will return this notification.
+
+Command::
++
+[source, cypher]
+----
+CREATE DATABASE foo TOPOLOGY 2 PRIMARIES
+----
+
+Description of the returned code::
+The following server(s) are cordoned `123e4567-e89b-12d3-a456-426614174000`. This can impact allocation decisions.
```

Comment:
Regarding INFORMATION vs. WARNING our rule of thumb is something like
WARNING = there is probably an issue and we can give you a suggestion of how you can improve
INFORMATION = there may be an issue but we are not sure and we cannot give you concrete advice but this might be good to know

I think we should include this (with a bit more professional phrasing maybe... ) in our guidelines @renetapopova . 

I am not sure if this case falls in the INFORMATION or WARNING bucket, but if we keep it at WARNING, it would be good to add a sentence of how you 'un-cordon' the server to the description.

Reply:
I chatted with the team about this and given these guidelines we feel it is more INFORMATION than WARNING. Will update the PR accordingly.